### ISSUE TITLE
Custom element doesn't exist: weatherbit-weather-card. type: 'custom:weatherbit-weather-card'

### ISSUE THREAD

**Issue by jayscovill** [Anwen747/weatherbit-weather-card#1]
I'm running 112.2 with HassOS 4.11 and have followed the instructions to include this custom card but I'm still getting "Custom element doesn't exist: weatherbit-weather-card. type: 'custom:weatherbit-weather-card'" when trying to create a custom card for this.

I'm sure I'm doing something wrong but I can't figure out what it is...I also had this same problem with the original custom weather card as well.

I've created the www/lovelace folder and put weatherbit-weather-card.js in it.
I've also create the www/icons/weather_icons directories and put the icons static and animated icon directories there.
I've added the /local/lovelace/weatherbit-weather-card.js line to Lovelace Dashboards -> Resources

Thanks!

**Comment by Anwen747** [comment]
It sounds like you've done everything right.  I use YAML mode and not Lovelace Dashboards, but as long as it shows up there as type module that should work.   I've had similar issues with the original custom weather card.  Try clearing cache from the browser developer tools - in Chrome, press F12, and then right-click on the refresh button and choose Empty Cache and Hard Reload.  It should be similar in other browsers.  You may even need to do it on every device that has the browser open.  Does that help?

**Comment by jayscovill** [comment]
Thanks for the suggestion but no dice...even in a completely different browser it still doesn't load the card.  Wondering if something is broken with HASS trying to manually add resources like that.

**Comment by Anwen747** [comment]
Have you tried restarting Home Assistant?

**Comment by jayscovill** [comment]
I have several times.  Interesting though.  It seemed to be a pathing problem to the .js file as there was a log error saying it couldn't find the file.  

The relative path for my community related stuff is /hacsfiles which points to /config/www.

I've corrected the file path to be where the error was expecting it but it's now throwing a different error:

http://hassio:8123/hacsfiles/weatherbit-weather-card/weatherbit-weather-card.js:6:1 Uncaught SyntaxError: Unexpected token '<'

Edit: Ok I know what's was causing the syntax error...my .js file got mangled when I downloaded it from github.  I've uploaded the unmangled copy! A total ID10T/Layer 8 error.  

We can close this one off since any problems I have won't be related to this one.  Thanks for your help!
